### PR TITLE
ACC-544 Migrate to jest

### PR DESCRIPTION
**Pull request by thurstontye**
Switch test suite to Jest prior to writing any real test for this component

### CODE REVIEW

File: test/src/js/get-user-status.spec.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-const { expect } = require('chai');
+
 
 const underTest = require('../../../src/js/get-user-status');
 
 describe('./src/js/get-user-status', function () {
-	it('getUserStatus should be a Function', function () {
-		expect(underTest).to.be.an('function');
+	test('getUserStatus should be a Function', function () {
```
Comment (AniaMakes) [inline test/src/js/get-user-status.spec.js:8]: Test title "should be function" does not match expectation of an object

File: test/src/js/config.spec.js [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
 
-const { expect } = require('chai');
-
 const underTest = require('../../../src/js/config');
 
 describe('./src/js/config', function () {
-	it('config should be an Object', function () {
-		expect(typeof underTest).to.equal('object');
+	test('config should be an Object', function () {
+		expect(typeof underTest).toEqual('object');
```
Comment (AniaMakes) [inline test/src/js/config.spec.js:7]: All other tests are `toBe` variety. Is there a reason this is a `toEqual`?

File: test/src/js/index.spec.js [hunk 3]
```diff
@@ -1,30 +1,25 @@
...
 
-	it.skip('should return undefined if the syndication flag is off and not call stubs', () => { });
-
-	context('when syndication customer', () => {
-		it.skip('should return undefined if the user isn’t migrated', () => { });
-		it.skip('should initialise the republising navigation', () => { });
-		it.skip('should end if the user isn’t allowed ft.com republishing', () => { });
-		it.skip('should initialise the rest if allowed ft.com republishing', () => { });
-	});
+	test.skip('should return undefined if the syndication flag is off and not call stubs', () => { });
```
Comment (AniaMakes) [inline test/src/js/index.spec.js:12]: Am I correct that this is "those will need to be written later, but not really part of this ticket"?

**Review by AniaMakes — COMMENTED** [review]
Just a few comments I'd love answered before approving, but overall LGTM!

File: src/js/util.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ function getContentIDFromHTMLElement (el) {
 				}
 
 				if (anchorEl && anchorEl.hasAttribute('href')) {
-					id = anchorEl.getAttribute('href').split('/').pop();
+					id = anchorEl.getAttribute('href').spltest('/').pop();
```
Comment (tolujimoh) [inline src/js/util.js:45]: Syntax Error from find and replace 
```suggestion
					id = anchorEl.getAttribute('href').split('/').pop();
```

File: test/src/js/index.spec.js (hunk 3 above)
Comment (thurstontye) [reply test/src/js/index.spec.js:12]: Yes. It's one of the reasons I got into this yak shaving excercise. There is no coverage on the init functionality so tricky making changes without writing test for existing features first. Thought I might as well switch up to jest if I'm gonna do that.

File: src/js/util.js (hunk 4 above)
Comment (thurstontye) [reply src/js/util.js:45]: noted

File: test/src/js/config.spec.js (hunk 2 above)
Comment (thurstontye) [reply test/src/js/config.spec.js:7]: 👍

File: test/src/js/get-user-status.spec.js (hunk 1 above)
Comment (thurstontye) [reply test/src/js/get-user-status.spec.js:8]: Fair comment but most of these tests are of no value other than placeholders right now

**Review by tolujimoh — APPROVED** [review]
LGTM 🚀

**Pull request merged**